src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql

```diff
@@ -851,6 +855,8 @@ define ReportNonOpioidMedicationRequests:
     Name:  ConceptText(O.medication as FHIR.CodeableConcept),
     Start: DateTimeText(O.authoredOn),
     End:   null as String, // MedicationRequest end date not supported in R4,
+    //Amount: QuantityText(O.value as FHIR.Quantity),
+    Quantity: O.dispenseRequest.quantity,
```

Comment:
same as my previous comment, maybe it is `O.dispenseRequest.quantity.value`, based on the FHIR json: `  "dispenseRequest": {
          "extension": [
            {
              "url": "http://cosri.org/fhir/pharmacy_name",
              "valueString": "REAL CHAIN STORE 1 (CO)"
            },
            {
              "url": "http://cosri.org/fhir/last_fill",
              "valueDate": "2020-03-28"
            }
          ],
          "quantity": {
            "value": 30
          },
          "expectedSupplyDuration": {
            "value": 8,
            "unit": "days",
            "system": "http://unitsofmeasure.org",
            "code": "d"
          }
        }`

Reply:
@mldorr Maggie, I think as I mentioned, this is correct: **`Quantity: O.dispenseRequest.quantity.value.value`** I re-build cql and the resulting json yields:
![Screen Shot 2020-08-11 at 9 44 54 AM](https://user-images.githubusercontent.com/12942714/89924778-57fe4080-dbb7-11ea-965b-f5b0d37a2435.png)